tests/test_transformations.py

```diff
@@ -314,3 +315,59 @@ def test_register_dependent_twice():
     set_dependents = transform.get_dependents()
 
     assert len(set_dependents) == 1
+
+
+def test_can_get_translation_as_4_by_4_matrix():
```

Comment:
Can you add a test for passing a transformation that isn't rotation/translation? I'm not sure what should happen in this case? I guess nothing.